.env.sample

```diff
@@ -0,0 +1,2 @@
+GITHUB_TOKEN=
+OPENAI_API_KEY=
```

Comment:
Ensure there is a documentation note indicating how to set these environment variables, as their absence may lead to confusion for new developers or users. Consider adding a README section addressing this.